(DOCSP-26857) Data API Can Evaluate Expresssions (#263)
mongodb/docs-app-services: source/data-api/custom-endpoints.txt

authorization information in the request. You define the authorization
scheme for each custom endpoint by configuring the endpoint function.

Endpoints natively support a set of built-in authorization schemes
that use a secret string to prove that the request is authorized. You

> can also define a custom authorization scheme that can replace or
> supplement the built-in schemes.

Reviewer: i find the phrase "replace or supplement" vague. perhaps:

```suggestion
can also define a custom authorization scheme that
you can use together with or instead of the built-in schemes.
```

      To learn how to sign your requests, see :ref:`Authorize the Request <endpoint-authorize>`.

Custom Authorization Schemes
++++++++++++++++++++++++++++

You can define a custom authorization :ref:`expression <expressions>` to
determine if an incoming authenticated request is allowed to run. The
expression is evaluated for each request and must evaluate to ``true``
to allow the request. If the expression evaluates to ``false``, the
request is not authorized and fails with an error. Requests that fail

> authorization are not counted toward your App's billed usage.

Reviewer: would be good to link to the [`%%request` expansion](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/data-api-can-evaluate/rules/expressions/#request-expansions) somewhere in this description.
